pyproject.toml

```diff
@@ -134,6 +134,7 @@ include = ["/README.md", "/Makefile", "/tests"]
 [tool.ruff]
 line-length = 120
 target-version = "py310"
+preview = true  # For `PLW1514`
```

Comment:
Can be set in `[tool.ruff.lint]`, so that you don't need to override it in `format`.

Reply:
Updated, thank you!
Didn't think about it